### PR TITLE
Login screen

### LINKED ISSUE #32: Login Screen
# As a user,
I want to be able to login,
so that I'm able to access my account.

# Description
Login Screen, Navigation from Login Screen and password retrieval.

# Done
- [ ] Login Screen

### PR DESCRIPTION
**Pull request by mschmauder-hub**
Fix #32 

Add login Screen. I have not yet added the handling of the login and navigation at this point. 
I thought this PR might get a bit big, so I'm going to split it into 2 Pull Requests.
Login Screen

### CODE REVIEW

File: SeriesMaster-Frontend/App/components/AppButton.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import styled from "styled-components/native";
+import colors from "../config/colors";
+import PropTypes from "prop-types";
+
+const ButtonContainer = styled.TouchableOpacity`
+  width: 50%;
+  height: 50px;
```
Comment (LebkuchenCJ) [inline SeriesMaster-Frontend/App/components/AppButton.js:8]: Can you use responsive sizes here, too?

File: SeriesMaster-Frontend/App/components/AppInputText.js [hunk 2]
```diff
@@ -1,13 +1,17 @@
 import React from "react";
 import styled from "styled-components/native";
 import PropTypes from "prop-types";
+import { MaterialCommunityIcons } from "@expo/vector-icons";
+import colors from "../config/colors";
 
 const InputContainer = styled.View`
   background: #fcfcfc;
```
Comment (LebkuchenCJ) [inline SeriesMaster-Frontend/App/components/AppInputText.js]: You are using theming on all colors, why not here?

**Review by LebkuchenCJ — APPROVED** [review]
GJ 👍 
only two small things.

File: SeriesMaster-Frontend/App/components/AppInputText.js (hunk 2 above)
Comment (mschmauder-hub) [reply SeriesMaster-Frontend/App/components/AppInputText.js]: good point!

**Pull request merged**